src/c20_server/flask_server.py

```diff
@@ -45,15 +43,38 @@ def _report_failure():
         user = User(user_id)
         job_manager.report_failure(user)
         return {}, 200
+
     return app
 
 
-if __name__ == '__main__':
+def update_job_manager(job_manager, client_data):
+    json_data = json.dumps(client_data)
+    job_list = handle_jobs(json_data)
+    print()
+    for job in job_list:
+        job_manager.add_job(job)
+        print('Adding Job To Job Manager...')
+        print(job, '\n')
+
+
+def save_data(data_repository, list_of_data_dicts):
+    data_items = DataExtractor.extract(list_of_data_dicts)
+    for data_item in data_items:
+        data_repository.save_data(data_item.folder_name,
+                                  data_item.file_name, data_item.contents)
+
+
+def launch():
```

Comment:
I did about the same idea in my repo #94, named redis_connect() that also has a test to its logic. So I think launch() and redis-connect() should be complied together in (near) future.

Reply:
Agreed, but let's do that in a different PR.